website/src/pages/index.tsx

```diff
@@ -27,7 +27,7 @@ function HomepageHeader() {
                   to="/doc">
                   Documentation
               </Link>
-
+              <Releases />
```

Comment:
Please add a `Download` menu for the release download, not the `index.tsx`.
```suggestion
              <Releases />
```

Reply:
![image](https://user-images.githubusercontent.com/172204/188340783-c785a2a3-3169-41d8-9c5f-39096242eef0.png)